wandb/apis/public.py

```diff
@@ -4425,6 +4425,62 @@ def load(self):
         self._attrs = response["project"]["artifactType"]["artifactCollection"]
         return self._attrs
 
+    @normalize_exceptions
+    def is_sequence(self) -> bool:
+        """Return True if this is a sequence."""
+        query = gql(
+            """
+            query CollectionType($entity: String!, $project: String!, $collection: String!) {
+                project(name: $project, entityName: $entity) {
+                    artifactCollection(name: $collection) {
```

Comment:
Hm. Can this query succeed running against such a server? If not, should I modify the query or should I just put it into a try/except and raise an error message saying the server doesn't support this operation?

Reply:
https://github.com/wandb/wandb/blob/34cb54e2187ff892747ce82fc4b0a3d3832a0c21/wandb/apis/public.py#L4465-L4490
I've seen instances like this in the public api code where they just format the query to use `artifactSequence` instead of `artifactCollection` based on the server version

Reply:
Okay, I changed to always using `artifactSequence`, since that's really what I'm looking for anyway. I do have to check to see if anything gets returned at all, but it seems to work.